tests/rpc_test.toit

```diff
@@ -216,6 +217,41 @@ test_blocking myself/int broker/RpcBroker tasks/int [test] -> none:
   broker.unregister_procedure name
   expect.expect_throw "No such procedure registered: 800": rpc.invoke myself name []
 
+test_sequential myself/int broker/RpcBroker -> none:
+  tasks ::= 10
+  name ::= 800
+  latches ::= {:}
+  concurrency := 0
+  broker.register_procedure name:: | args |
+    result := null
+    try:
+      concurrency++
+      expect.expect_equals 1 concurrency  // Should be sequential!
```

Comment:
wouldn't it be better with a `yield` here to allow the other tasks to jump in?

Reply:
The last.get call blocks until explicitly told to continue from the outside. That was intended to be (even) more robust than yielding.

Reply:
Perfect.